### DISCUSSION TITLE
Orange and teal colors rae not displaying in tailwind css

### DISCUSSION THREAD

**Discussion by khansawidhani** [tailwindlabs/tailwindcss#4248] [Help]
Hi experts, I need your help. I am using tailwindcss in my project. but problem is that only orange and teal colors are not displaying on the web page result. All other colors are showing properly. I have copied sample code from the documentation and tried but still, the orange color did not display. 
Tailwind IntelliSense is also detecting colors in my HTML file but not on the web page.
Does anyone else face this issue?
If anyone has the solution, please do let me know. 
let me show my tailwind.config.css file  

```const colors = require('tailwindcss/colors');
module.exports = {
  purge: [],
  darkMode: false, // or 'media' or 'class'
  theme: {
    colors: {
      transparent: 'transparent',
      current: 'currentColor',
      black: colors.black,
      white: colors.white,
      gray: colors.trueGray,
      indigo: colors.indigo,
      red: colors.rose,
      yellow: colors.amber,
      orange: colors.orange,
      teal : colors.teal,
      


    },
    extend:{},
  },
  variants: {
    extend: {},
  },
  plugins: [],
}
```
index.html file 
```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Ecommerce App</title>

    <link rel="stylesheet" href="build/output.css">
</head>
<body>
    <h1 class="text-6xl text-gray-600">Hello World</h1>
    <button class="text-indigo-500 p-3">indigo 500</button>
    <button class="text-red-500 p-3">Red 500</button>
    <button class="text-yellow-500 p-3">yellow 500</button>
    <button class="text-orange-500 p-3">orange 500</button>
    <button class="text-teal-500 p-3">teal 500</button>
</body>
</html>```


see the screenshot of tailwind IntelliSense color detection in the HTML file
!(https://user-images.githubusercontent.com/57295367/117109077-881ced80-ad9d-11eb-8d93-624210dbeb4c.jpg)

my webpage result:
![image](https://user-images.githubusercontent.com/57295367/117109345-f2359280-ad9d-11eb-81c2-7b568cf4db7c.png)

**Comment by alexanderbluhm** [comment]
@khansawidhani Hi! Can you search for `teal` and `orange` classes in the `build/output.css`?

Nevertheless, you should enable PurgeCSS for smaller css files in production:
https://tailwindcss.com/docs/optimizing-for-production#removing-unused-css
This may also solve your problem.

Also please make sure to recompile your css file after you changed something in the config.

**Reply by khansawidhani** [reply]
yes. I have imported all colors from tailwind/colors.js.
Infact tailwind intelliSence is also detecting and showing colors in html file.
I can also search for orange and teal classes in Output.css
